searchlib/src/vespa/searchlib/tensor/distance_functions.h

```diff
@@ -163,6 +173,11 @@ class GeoDegreesDistance : public DistanceFunction {
         double hav_central_angle = hav_lat + cos(lat_A)*cos(lat_B)*hav_lon;
         return hav_central_angle;
     }
+    double convert_threshold(double threshold) const override {
+        double half_angle = threshold / (2 * 6371.0088);
```

Comment:
Consider adding a constexpr for the magic constant 6371.0088. Also used in to_rawscore().